CMS-Enterprise/mint-app
# [NOREF]mto solution and milestone facilitated by multiselect

# NOREF

## Description

Milestone and Solution facilitated by fields should be multi-select. This PR updates them to be.
A common Milestone will auto populate the facilitated by field when you create it.
ALSO, Solution poc fields should be nullable. This PR updates them.




## How to test this change

1.  Create a milestone from a common milestone
   a. make sure the result populates the facilitated by multi select
2. Make sure you can update the facilitated by field for a milestone
3. Make a solution
4. Make sure you can update the facilitated by field
5. Note when you make a solution from common solution, POC name and email are now both nullable.

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
